src/main/java/com/google/javascript/gents/RemoveGoogScopePass.java

```diff
@@ -73,11 +73,18 @@ private void rewriteGoogScope(Node n) {
 
     @Nullable Node nodeToMove = blockOfScopeContents.getFirstChild();
     while (nodeToMove != null) {
-      nodeToMove = maybeRewriteAlias(nodeToMove);
-      if (nodeToMove == null) {
-        break;
+      // After maybeRewriteAlias, nodeToMove is either (1)already detached or (2)needs to be moved.
+      @Nullable Node nodeToCheck = maybeRewriteAlias(nodeToMove);
+      // (1) nodeToMove is detached, alias to provided namespace is recorded, the next node is
+      // returned. The next node should be 'maybeRewriteAlias()' checked before it can be moved out
+      // of goog.scope
+      if (null != nodeToCheck) {
```

Comment:
Fixed.

I was under the impression that constants on the left are [better](http://www.moserware.com/2008/01/constants-on-left-are-better-but-this.html) but Java isn't vulnerable to this problem.